scripts/scripts.js

```diff
@@ -1363,13 +1364,15 @@ sampleRUM.drain('convert', (cevent, cvalueThunk, element, listenTo = []) => {
     try {
       // get all stored experiments from local storage (unified-decisioning-experiments)
       const experiments = JSON.parse(localStorage.getItem('unified-decisioning-experiments'));
-      Object.entries(experiments)
-        .map(([experiment, { treatment, date }]) => ({ experiment, treatment, date }))
-        .filter(({ date }) => Date.now() - new Date(date) < MAX_SESSION_LENGTH)
-        .forEach(({ experiment, treatment }) => {
-          // send conversion event for each experiment that has been seen by this visitor
-          sampleRUM('variant', { source: experiment, target: treatment });
-        });
+      if (experiments) {
```

Comment:
I don't think we really need this, given that we `catch` and discard the exception.

Reply:
without it, when there are no experiments the exception is raised, and the call to `sampleRUM('convert', { source: cevent, target: cvalue, element: celement });` never executed, as it is inside the `try`

Reply:
Ok.